sdk/anomalydetector/azure-ai-anomalydetector/samples/sample_multivariate_detect.py

```diff
@@ -19,200 +19,214 @@
     2) ANOMALY_DETECTOR_ENDPOINT - the endpoint to your source Anomaly Detector resource.
 """
 
+import json
 import os
 import time
 from datetime import datetime, timezone
 
 from azure.ai.anomalydetector import AnomalyDetectorClient
-from azure.ai.anomalydetector.models import DetectionRequest, ModelInfo, LastDetectionRequest
-from azure.ai.anomalydetector.models import ModelStatus, DetectionStatus
 from azure.core.credentials import AzureKeyCredential
 from azure.core.exceptions import HttpResponseError
 
 class MultivariateSample:
 
-    def __init__(self, subscription_key, anomaly_detector_endpoint, data_source=None):
+    def __init__(self, subscription_key, anomaly_detector_endpoint):
         self.sub_key = subscription_key
         self.end_point = anomaly_detector_endpoint
 
         # Create an Anomaly Detector client
 
         # <client>
-        self.ad_client = AnomalyDetectorClient(AzureKeyCredential(self.sub_key), self.end_point)
+        self.ad_client = AnomalyDetectorClient(self.end_point, AzureKeyCredential(self.sub_key))
         # </client>
 
-        self.data_source = data_source
+    def list_models(self):
 
-    def train(self, start_time, end_time):
+        # List models
+        skip = 0
 
-        # Number of models available now
-        model_list = list(self.ad_client.list_multivariate_model(skip=0, top=10000))
-        print("{:d} available models before training.".format(len(model_list)))
+        next_link = None
+        model_list = []
 
-        # Use sample data to train the model
-        print("Training new model...(it may take a few minutes)")
-        data_feed = ModelInfo(start_time=start_time, end_time=end_time, source=self.data_source)
-        response_header = \
-            self.ad_client.train_multivariate_model(data_feed, cls=lambda *args: [args[i] for i in range(len(args))])[
-                -1]
-        trained_model_id = response_header['Location'].split("/")[-1]
+        while next_link != '':
+            r = self.ad_client.list_multivariate_model(skip=skip, top=10000)
```

Comment:
not recommended to set 'top' to such a large number. we can set it to 10 as an example